draft-thomson-webpush-vapid.md

```diff
@@ -341,8 +341,8 @@ Credentials are invalid if:
   the creation of the push message.
 
 A push subscription that is not restricted to a particular key MAY still
-validate a token that is present, except for the last check.  A push service MAY
-then reject a request if the token is found to be invalid.
+validate a JWT that is present, with the exception of the last check.  A push
```

Comment:
I would suggesting eliminating the discussion of JWT validation. This section is about restricting subscriptions. The draft has already covered validation of the JWT earlier:

This JWT is included in an Authorization header field, using an authscheme
of "WebPush". A push service MAY reject a request with a 403
(Forbidden) status code [RFC7235] if the JWT signature or its claims
are invalid.